app/javascript/components/widget-controller.jsx

```diff
@@ -73,6 +75,20 @@ export const WidgetController = ({ client, cityName }) => {
           return <DisconnectedDisplay cityName={cityName} error={error} />;
         }
 
+        if (
+          R.pathOr(
+            data.location.widgets.enabled === [],
+            ['NullPointerhandler', 'condition'],
+            {
+              NullPointerhandler: {
+                conditon: data.location.widgets.enabled === [],
+              },
+            },
+          )
```

Comment:
`pathOr` is intended to accept the arguments: defaultValue, arrayPath, object. An example:
```
const something = { a: { b: { c: 6 } } };
console.log(pathOr(2, ['a', 'b', 'c'], something)); // 6
console.log(pathOr(2, ['a', 'b', 'd'], something)); // 2
console.log(pathOr(2, ['a', 'b', 'c'], null)); // 2
```
The purpose is to safely access values in an nested object that may or may not contain null/undefined. The code above can still have a null pointer exception if data is null.